pages/implementation/avo-tracking-plan-webhook.mdx

```diff
@@ -50,7 +50,8 @@ You can examine and download the exact JSON in the Publishing tab of your [Avo w
                     "name": "Mapped Property Name",
                     "destinationId": "all" | "anotherDestinationId"
```

Comment:
Why is it red 🤔

Reply:
Great question 👀  Maybe because this isn't a valid json? I think we should try to keep the example on a valid format, so people can easily copy it to play around with. Can we communicate this somehow differently, e.g. where we describe the interface?